### PR TITLE
matt hinea

### PR DESCRIPTION
**Pull request by matthinea**

### CODE REVIEW

**Comment by lynchd2** [comment]
Test

**Comment by lynchd2** [comment]
app/views/static_pages/about_edit.html.erb
-On line 20 redundant url: profile_path.. It is not needed

**Comment by lynchd2** [comment]
-Edit Profile button in /about url does not work.
-Some kind of validation if user does not input an appropriate birthday format, or break it up into drop down selects. Otherwise user keeps clicking the button and nothing happens.
-Tried entering year 0, and website crashed.
-Entering a letters in the telephone number field results in a crash 
-Disable turbolinks

**Comment by lynchd2** [comment]
-User creation w/ same email. No unique constraint.
-About me and words to live by text areas can be made bigger and smaller. Try resizing them with cols:50 and rows:50, and in css add an attribute "resize:none;" so that they cannot resize it.
-No way to go to other users' profiles.
-Search bar does not work.

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+  private 
+
+  def whitelisted_params
+    params.require(:comment).permit(:id, :body)
+  end
+
+end
```
Comment (philipcolejohnson) [inline app/controllers/comments_controller.rb:40]: Great! Like your short helper methods

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    if @post.save
+      flash[:success] = "Your post has been created."
+      redirect_to user_timeline_path(current_user)
+    else
+      flash.now[:error] = @post.errors.full_messages.join(' ')
+      render :index
+    end
+  end
+
+  def destroy
```
Comment (philipcolejohnson) [inline app/controllers/posts_controller.rb]: I could delete another person's posts!

File: app/views/static_pages/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,96 @@
+<!-- replace static content with content from @user's profile, except birth_date -->
+<div id="about" class="container">
+  <header class="row" id="row-header">
+    <div class="col-xs-2 col-xs-offset-5 content-center">
+      <h1 id="about-header">About</h1>  
+    </div>
+    <div class="col-xs-2 col-xs-offset-3 content-center">
+      <input type="button" id="edit-button" value="Edit your Profile">
```
Comment (philipcolejohnson) [inline app/views/static_pages/about.html.erb:8]: This button isn't working

**Pull request closed without merging**